### DISCUSSION TITLE
In-game leaderboard should display max combo achieved instead of current combo count

### DISCUSSION THREAD

**Discussion by ikkue** [ppy/osu#20532] [Ideas] [answered]
As in osu!(stable), the in-game leaderboard should display the player's highest achieved combo chain rather than the current combo like in multiplayer.

I acknowledge that it's only the initial implementation, but I think it's good to open a discussion on it anyways.

### osu!(stable)
![screenshot012](https://user-images.githubusercontent.com/11213135/193045928-6acea2e8-00d7-456b-b383-a89e116b1ea0.png)

### osu!(lazer)
![osu_2022-09-29_20-31-00](https://user-images.githubusercontent.com/11213135/193046141-ca9e79d6-a91d-4398-9e8b-41fb2a134973.png)

**Comment by ghost** [comment, marked as answer]
(not too hard of a tweak to make either, it's a one-line change)

**Reply by nanashi-1** [reply]
tweaked it in #20547 

hope it gets merged
